cogs/Memes.py

```diff
@@ -18,6 +18,12 @@ async def onlyfans(self, inter):
         await inter.send(
             "https://media.wired.com/photos/59548ac98e8cc150fa8ec379/master/w_2560%2Cc_limit/GettyImages-56196238.jpg"
         )
+    @commands.slash_command(name="boners with lewis", description="Boners with lewis")
```

Comment:
Spaces won't work in the name

Reply:
I think it does work, because the tag commands have a space in them

Reply:
Those are subcommands

Reply:
Oh okay didn't know that